### PR TITLE
Add accordion plugin files

### PR DESCRIPTION
**Pull request by Xenorith**
and fix accordion blocks in EMR

### CODE REVIEW

**Comment by Xenorith** [comment]
sample screenshot

Screenshot 2019-12-05 19 46 57

File: docs/_plugins/collapsible.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+        site = context.registers[:site]
+        converter = site.find_converter_instance(::Jekyll::Converters::Markdown)
+        content = converter.convert(super)
+
+        output = <<~EOS
+          <div class="card" style="overflow:visible">
+            <div class="card-header" id="#{headingID}" style="padding:0px">
+              <h4 class="mb-0">
+                <button class="btn btn-link collapsed" data-toggle="collapse" data-target="##{collapsedID}" aria-expanded="false" aria-controls="#{collapsedID}">
```
Comment (ZacBlanco) [inline docs/_plugins/collapsible.rb]: is `##` expected or a typo?

Comment (Xenorith) [reply docs/_plugins/collapsible.rb]: it is expected. i am not sure how this all works, but `#{varName}` generates to `varValue` in HTML, while `##{varName}` generates to `#varValue`. the use of `#` in HTML appears for the `data-target` and `data-parent` values; when i remove it from the HTML, the accordion blocks don't expand/collapse

Comment (ZacBlanco) [reply docs/_plugins/collapsible.rb]: sounds good. Thanks for the explanation

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Pull request merged**